src/ageml/messages.py

```diff
@@ -73,6 +73,14 @@
     "https://github.com/compneurobilbao/ageml/blob/main/docs/input_file_specification.md#clinical-file"
 )
 
+covcorr_mode_long_description = (
```

Comment:
Missing description of what is the default if covariates files given but this flag is not.

Reply:
True, just added it.